next.config.mjs

```diff
@@ -0,0 +1,31 @@
+import nextra from 'nextra';
+
+import getNextData from './next.data.mjs';
+
+const withNextra = nextra({
+  theme: './theme.tsx',
```

Comment:
```suggestion
  theme: 'theme.tsx',
```
I think this will fix the windows CI issue (at least it builds now on my fork with this change)

Reply:
Thanks for the fix, @mikeesto! That was actually really simple!

Reply:
Oh wow, makes a lot of sense! Thank you!